sys/include/fd.h

```diff
@@ -36,11 +36,11 @@ typedef struct {
     /** private status of this fd_t */
     int __active;
 
-    /** the internal filedescriptor */
-    int fd;
+    /** Stores the RIOT internal value for the file descriptor (not POSIX). */
+    int internal_fd;
```

Comment:
What was it with underscores? (https://github.com/RIOT-OS/RIOT/pull/2355#discussion_r23561946):

> Leading underscores are not really recommended in include guard macro names.
> 
> http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf
>
>> 7.1.3 Reserved identifiers
>> All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.